crates/bevy_render/src/primitives/mod.rs

```diff
@@ -131,15 +131,15 @@ impl Plane {
 #[reflect(Component)]
 pub struct Frustum {
```

Comment:
I see the doc string for `Frustrum` mention planes. Could you reword it as well?